### PR TITLE
Remove all HTML templates, start to use RestPlus to check the code.

### PR DESCRIPTION
**Pull request by zfrank7777**

### CODE REVIEW

**Comment by amjltc295** [comment]
Requirement.txt should be updated. flask_restplus is missing.

File: src/surfing_penguin/forms.py [hunk 1]
```diff
@@ -1,16 +0,0 @@
-"""forms.py: This file contains classes that respectively represent a form."""
```
Comment (amjltc295) [inline src/surfing_penguin/forms.py]: Why delete this file?

**Comment by amjltc295** [comment]
qstnr_id in GET /fill/{qstnr_id}. is not corresponding to GET /select.

File: src/surfing_penguin/routes.py [hunk 2]
```diff
@@ -1,51 +1,54 @@
...
+    def get(self):
+        return "Welcome to Surfing Penguin"
+
+
+@api.route('/login')
+class login(Resource):
+    def get(self):
+        return "Please Login"
+
+    def put(self):
```
Comment (amjltc295) [inline src/surfing_penguin/routes.py]: Why is this PUT, not POST?

**Review by amjltc295 — COMMENTED** [review]
I think you should still keep some template for back end management, e.g. account and survey management for manager.

**Comment by zfrank7777** [comment]
The problem with qstnr_id will be handled when implementing database.

File: src/surfing_penguin/routes.py [hunk 3]
```diff
@@ -4,6 +4,7 @@
 from flask import render_template, flash, redirect, url_for, request
 from flask_restplus import Resource
 from surfing_penguin.forms import LoginForm
+from surfing_penguin.models import Qstnr, question
```
Comment (amjltc295) [inline src/surfing_penguin/routes.py]: These lines did not pass the flake8 check.

File: src/surfing_penguin/models.py [hunk 4]
```diff
@@ -1,6 +1,36 @@
...
-from surfing_penguin import db
+from surfing_penguin import db, api
+from flask_restplus import fields
+
+
+question = api.model("question_model", {
+        'content': fields.String,
+        'id': fields.Integer
+    })
+
```
Comment (amjltc295) [inline src/surfing_penguin/models.py]: Why do you put the API model here? It should be in routes.py or other files. models.py is for the database. 
Also, you should add description for the API model.

File: src/surfing_penguin/routes.py (hunk 3 above)
Comment (amjltc295) [reply src/surfing_penguin/routes.py]: Please make sure the code could work and pass the style checker before pushing. This is the basic request. surfing_penguin.forms  doesn't even exist.

File: src/surfing_penguin/routes.py [hunk 5]
```diff
@@ -18,6 +19,16 @@
         'id': 1
     }
 ]
```
Comment (amjltc295) [inline src/surfing_penguin/routes.py]: For testing, you should add some unit tests. See https://docs.pytest.org/en/latest/

**Review by amjltc295 — CHANGES_REQUESTED** [review]
A lot to be improved.

File: src/surfing_penguin/models.py (hunk 4 above)
Comment (zfrank7777) [reply src/surfing_penguin/models.py]: I think the Qstnr class will be recorded into the database, and it needs the api.model.
Or it should not work this way?

File: src/surfing_penguin/db_interface.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
+from surfing_penguin import api
+from flask_restplus import fields
+
+question = api.model("question_model", {
```
Comment (amjltc295) [inline src/surfing_penguin/db_interface.py]: This is the API model that should be put in routes.py

**Review by amjltc295 — CHANGES_REQUESTED** [review]

**Comment by amjltc295** [comment]
Lgtm

**Pull request merged**